core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -703,7 +703,7 @@ class EMConfig {
 
     @Experimental
     @Cfg("Enable Expectation Generation. If enabled, expectations will be generated. " +
-            "A variable called activeExpectations is added to each test case, with a default value of false. If set to true, an expectation that fails will cause the test case containing it to fail.")
+            "A variable called expectationsMasterSwitch is added to the test suite, with a default value of false. If set to true, an expectation that fails will cause the test case containing it to fail.")
     var expectationsActive = false
```

Comment:
are the e2e tests passing when this is set to `true`?

Reply:
Generated tests pass, yes.
EM's e2e tests fail (at least the TestCaseWriterTest does). I'll adjust that and see if anything else causes a problem.